app/presenters/foreman_tasks/execution_plan_presenter.rb

```diff
@@ -0,0 +1,40 @@
+module ForemanTasks
+  class ExecutionPlanPresenter
+
+    attr_accessor :task
+
+    def initialize(task)
+      @task = task
+    end
+
+    def plan
+      output = []
+      @execution_plan = @task.execution_plan
+      output << {'run' => process_flow(@execution_plan.run_flow)}
+      output << {'finalize' => process_flow(@execution_plan.finalize_flow)}
+      output
+    end
+
+    def process_flow(flow)
+      case flow
+      when ::Dynflow::Flows::Sequence
```

Comment:
The reason for not having this in foreman-tasks is to have the same format in foreman_tasks and smart_proxy_dynflow.